artifacts/flagger/crd.yaml

```diff
@@ -91,7 +91,7 @@ spec:
                   properties:
                     items:
                       type: object
-                      required: ['name', 'interval', 'threshold']
+                      required: ['name', 'threshold']
```

Comment:
Should this result in new API version?

Reply:
It's not a breaking change. I've stopped incrementing the API version if the new fields are optional.